Lombiq.Tests.UI/Extensions/EventsOrchardCoreUITestExecutorConfigurationExtensions.cs

```diff
@@ -27,38 +27,29 @@ static bool IsNoAlert(UITestContext context)
                 }
             }
 
-            IWebElement html = null;
+            PageNavigationState navigationState = null;
```

Comment:
So the logic here is the same just factored out, right?

Reply:
Yes.